lib/browser/api/browser-window.js

```diff
@@ -96,17 +95,6 @@ BrowserWindow.prototype._init = function () {
   // Notify the creation of the window.
   app.emit('browser-window-created', {}, this)
 
-  // Be compatible with old APIs.
```

Comment:
I'm good removing them, people might be complaining though since we didn't generate deprecation notices for them.